modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -249,17 +289,17 @@ and add the Maven facade as a repository in the `repositories` section.
     <repository>
       <id>Repository</id>
       <name>Corporate Repository</name>
-      <url>https://maven.anypoint.mulesoft.com/api/v1/organizations/ORG_ID/maven</url>
+      <url>https://maven.anypoint.mulesoft.com/api/v3/organizations/ORGANIZATION_ID/maven</url>
       <layout>default</layout>
     </repository>
   </repositories>
  ...
 </project>
 ----
 
-Replace ORG_ID with your organization ID.
+Replace ORGANIZATION_ID with your organization ID.
 
-You can use `+https://maven.anypoint.mulesoft.com/api/v1/maven+` as a valid URL for the `+<url>+` value, as this is the default to publish to the master organization. In the EU, use `+https://maven.eu1.anypoint.mulesoft.com/api/v1/maven+` as the URL.
+You can use `+https://maven.anypoint.mulesoft.com/api/v3/maven+` as a valid URL for the `+<url>+` value, as this is the default to publish to the master organization. In the EU, use `+https://maven.eu1.anypoint.mulesoft.com/api/v3/maven+` as the URL.
```

Comment:
internationalization:
```suggestion
You can use `+https://maven.anypoint.mulesoft.com/api/v3/maven+` as a valid URL for the `+<url>+` value, because this is the default to publish to the master organization. In the EU, use `+https://maven.eu1.anypoint.mulesoft.com/api/v3/maven+` as the URL.
```